unit-tests-java8/src/test/java/org/eclipse/collections/test/set/SetTestCase.java

```diff
@@ -105,6 +108,7 @@ default void Collection_add()
     @Test
     default void Collection_size()
     {
+        CollectionTestCase.super.Collection_size();
```

Comment:
The three missing calls to super were old bugs. I only noticed this because I was looking around for overrides of remove. The assertions from the superclass do apply and do pass.